### ISSUE TITLE
Focus indicator does not appear correctly on <a> elements wrapping on multiple lines in Chrome, Edge

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_base#335]
Examples:

![image](https://user-images.githubusercontent.com/19519457/210758914-4dba9361-3e6b-4fea-bc2d-97c66fa11726.png)
![image](https://user-images.githubusercontent.com/19519457/210758963-8e373665-0df7-4b0d-8db7-e2a0903babfd.png)
![image](https://user-images.githubusercontent.com/19519457/210759055-b441ce3b-995f-4a7f-bccb-3716d5cce6e5.png)

I think this is a WCAG 2.1 AA fail because without the underline, the focus indicator does not have sufficient contrast against white backgrounds.

Correct behaviour (working in Safari):

![image](https://user-images.githubusercontent.com/19519457/210759427-2f58ba3f-fda3-4ee0-bc0a-243fa61a64f8.png)
